<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vatek_romtool</class>
 <widget class="QMainWindow" name="vatek_romtool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>484</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>vatek_romtool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
    <item>
     <widget class="qtvUIListStorage" name="listorage">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stcontext">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionDevice"/>
    <addaction name="actionImage"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionUpdateDevice"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionUpdate_Image">
   <property name="text">
    <string>Update Image</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionUpdate_Image_2">
   <property name="text">
    <string>Update Image</string>
   </property>
  </action>
  <action name="actionUpdateImage">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDevice">
   <property name="text">
    <string>Device</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionUpdateDevice">
   <property name="text">
    <string>Write Image</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtvUIListStorage</class>
   <extends>QListWidget</extends>
   <header>ui/base/qtv_ui_storage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
